.env.example

```diff
@@ -1,6 +1,6 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:HYL7ToFdCupKbIccJB/ANQUzWkpm5bZApPc0a+awcJY=
```

Comment:
this is the secret data for the application. Don't put them in the repository. You can generate this data on your local computer